source/Hardpoint.cpp

```diff
@@ -457,7 +457,7 @@ void Hardpoint::Fire(Ship &ship, const Point &start, const Angle &aim)
 	// Anti-missile sounds can be specified either in the outfit itself or in
 	// the effect they create.
 	if(outfit->WeaponSound())
-		Audio::Play(outfit->WeaponSound(), start);
+		Audio::Play(outfit->WeaponSound(), start, IsSpecial() ? SoundCategory::ANTI_MISSILE : SoundCategory::WEAPON);
```

Comment:
This check would also be true for tractor beams. Is this intentional?

Reply:
Yes. There aren't enough tractor beams to be worth separating them for now.